pkg/operator/operator.go

```diff
@@ -158,6 +162,7 @@ func (op *Operator) Configure() error {
 func (op *Operator) setupWorkloadInformers() {
 	deploymentInformer := op.kubeInformerFactory.Apps().V1beta1().Deployments().Informer()
 	op.addEventHandlers(deploymentInformer, apps.SchemeGroupVersion.WithKind("Deployment"))
+	deploymentInformer.AddEventHandler(op.extractDockerLabel.ExtractDockerLabelHandler())
```

Comment:
This needs to support all workload types.

- Deployment
- ReplicaSet
- ReplicationContorller
- DaemonSet
- StatefulSet
- Job